main.go

```diff
@@ -192,11 +200,16 @@ func writeCachePullTimestamp() (err error) {
 }
 
 func main() {
+	// Possible values of GOARCH: https://go.dev/doc/install/source#environment
+	const currentArchitecture = runtime.GOARCH
```

Comment:
I think Rosetta on macOS will not affect this but just want to double check with you. 

We will never run the bitrise cli (or the steps) through Rosetta so this `GOARCH` will always have the correct value, right?

Reply:
might be worthing doing some validation and having a soft warning
```
sysctl -n sysctl.proc_translated // return 0
arch -x86_64 sysctl -n sysctl.proc_translated // return 1 in Rosetta
```